en/release-notes/upgrade-guide-v3.0.md

```diff
@@ -54,56 +48,166 @@ Please get the `UITransform` component on the node first, and then use the corre
 
-#### Resource thumbnail panel
+#### Asset Preview Panel
 
-Select the resource in the resource manager to display resource thumbnails in the resource thumbnail panel for easy viewing:
+Select the resource in the __Asset__ panel to display resource thumbnails in the __Asset Preview__ panel. If the folder where the resource is located is selected, the thumbnails of all resources under the folder can be displayed for easy viewing.
 
-![image](https://user-images.githubusercontent.com/1503156/100602913-78f1c980-333f-11eb-9f9a-18e214548ce7.png)
+![image](assets-preview.png)
 
-#### Animation editor upgrades
+#### Animation Panel Upgrade
 
 - Support the search and display filtering of nodes in the node tree panel.
 - Support using the system clipboard to copy and paste all animation data (nodes, tracks, and key frames) on nodes.
 - Support multi-select nodes to add attribute tracks in batches.
-- Optimize the experience of selecting and deselecting key frames (Ctrl + mouse click to select key frames to deselect them).
-- After selecting multiple key frames, continue to click to edit the curve or select unselected key frames.
+- Optimize the experience of selecting and deselecting key frames (__Ctrl + mouse click__ to select key frames to deselect them).
 - Support continuing to edit node attributes in the animation editing state, including particle and model material attributes, etc.
 
-![image](https://user-images.githubusercontent.com/1503156/100603114-a2aaf080-333f-11eb-8bd3-0997721adcb6.png)
+![image](animation.png)
 
-#### Project settings panel update
+#### Project Setting Panel Update
 
-It is divided into three parts: **Engine module**, **project settings**, and **compressed texture configurations**.
+It is divided into three parts: __Engine Manager__, __Project Setting__, and __Build__.
 
 The Physics Collision Group uses the `PhysicsSystem.PhysicsGroup` type independently, and no longer shares the group configuration with `Node.Layers`:
 
-![image](https://user-images.githubusercontent.com/1503156/100603220-be15fb80-333f-11eb-9ddd-a6b97455c468.png)
+![image](project-setting.png)
+
+The __Texture Compression__ is modified to configure the preset in the __Project Setting__ panel, then select the image resource in the __Asset__ panel, and then select the preset method.<br>
+After the old project is upgraded, the editor will automatically scan all the compressed texture configurations in the project and sort out several presets. Since it is automatically scanned, the generated name may not be what you want, you can modify it here.
+
+![image](texture-compress-setting.png)
+
+### Build Directory Differences
+
+Cocos Creator 2.x and Cocos Creator 3.0 differ to a certain extent in the directories generated after building on different platforms. Let's take __v2.4.3__ as an example and compare it with __v3.0__ on Web, Native and WeChat Mini Game platforms respectively.
```

Comment:
Cocos Creator 2.x and Cocos Creator 3.0
-->
__Cocos Creator 2.x__ and __Cocos Creator 3.0__